manifests/client/deployment.yaml

```diff
@@ -74,11 +80,8 @@ spec:
         - containerPort: 8443
           name: https
         resources:
-          limits:
-            cpu: 20m
-            memory: 40Mi
           requests:
-            cpu: 10m
+            cpu: 1m
```

Comment:
is this enough?

Reply:
for kube-rbac-proxy, yes, i measured it on a local e2e prometheus dump

Reply:
ack, looks like this was set as the request for every container in the pod. Even thought it's just a request, I don't think it's enough for telemeter client